doc/api/vm.md

```diff
@@ -866,6 +866,36 @@ local scope, so the value `localVar` is changed. In this way
 `vm.runInThisContext()` is much like an [indirect `eval()` call][], e.g.
 `(0,eval)('code')`.
 
+## vm.compileFunction
```

Comment:
1. It seems this should go before `## vm.createContext([sandbox[, options]])` ABC-wise.
2. The signature seems to be `vm.compileFunction(code, params, options)` if all parameters are mandatory (if not, they can be wrapped in `[]`).